content/token-program.md

```diff
@@ -59,7 +59,7 @@ const tokenMint = await createMint(
 - `freezeAuthority` - an account authorized to freeze the tokens in a token account. If freezing is not a desired attribute, the parameter can be set to null
 - `decimals` - specifies the desired decimal precision of the token
 
-When creating a new mint from a script that has access to your secret key, you can simply use the `createMint` function. However, if you were to build a website to allow users to create a new token mint, you would need to do so with the user's secret key without making them expose it to the browser. In that case, you would want to build and submit a transaction with the right instructions.
+When creating a new mint from a script that has access to your secret key, you can simply use the `createMint` function. You can check out the source code of `createMint` [here](https://github.com/solana-labs/solana-program-library/blob/48fbb5b7/token/js/src/actions/createMint.ts#L29). However, if you were to build a website to allow users to create a new token mint, you would need to do so with the user's secret key without making them expose it to the browser. In that case, you would want to build and submit a transaction with the right instructions.
```

Comment:
Documentation links are good, but I don't think we should refer to users to the source code - if we're asking users to read the source, that means our documentation has failed.